### DISCUSSION TITLE
Analyzers version 17.0.63 is not available

### DISCUSSION THREAD

**Discussion by siewers** [microsoft/vs-threading#934] [Q&A]
On NuGet, version 17.0.63 is released and points to this tag https://github.com/microsoft/vs-threading/releases/tag/v17.0.63, but it doesn't exist.

**Comment by AArnott** [comment]
Good catch. I've published the release.

**Comment by KarlWindhager** [comment]
I think its the same again with 17.0.64: https://github.com/microsoft/vs-threading/releases/tag/v17.0.64

**Reply by AArnott** [reply]
Fixed.
